src/videojs-http-streaming.js

```diff
@@ -140,18 +141,56 @@ Hls.canPlaySource = function() {
     'your player\'s techOrder.');
 };
 
-const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
+const emeKeySystems = (keySystemOptions, mainSegmentLoader, audioSegmentLoader) => {
   if (!keySystemOptions) {
     return keySystemOptions;
   }
 
+  let videoMimeAndCodec;
```

Comment:
should these just be `videoMimeType` and `audioMimeType`? seems like that is they way that it is referred to in most of the code